src/main/twirl/gitbucket/core/pulls/proposals.scala.html

```diff
@@ -7,7 +7,7 @@
     <div class="box-content" style="line-height: 20pt; margin-bottom: 6px; padding: 10px 6px 10px 10px; background-color: #fff9ea">
       <strong><i class="menu-icon octicon octicon-git-branch"></i><span class="muted">@branch</span></strong>
       <a class="pull-right btn btn-success" style="position: relative; top: -4px;"
-         href="@helpers.url(repository)/compare/@{parent.owner}:@{helpers.urlEncode(parent.repository.defaultBranch)}...@{repository.owner}:@{helpers.urlEncode(branch)}">Compare & pull request</a>
+         href="@helpers.url(repository)/compare/@{parent.owner}:@{helpers.helpers.encodeRefname(parent.repository.defaultBranch)}...@{repository.owner}:@{helpers.encodeRefname(branch)}">Compare & pull request</a>
```

Comment:
Travis build error is here: https://travis-ci.org/gitbucket/gitbucket/builds/424075030#L539

I guess `helpers.helpers.encodeRefname` should be `helpers.encodeRefname`.